dataactbroker/handlers/fileHandler.py

```diff
@@ -397,14 +398,15 @@ def getStatus(self):
             submissionInfo["last_updated"] = submission.updated_at.strftime("%Y-%m-%dT%H:%M:%S")
 
             for job_id in jobs:
+                job = sess.query(Job).filter_by(job_id = job_id).one()
```

Comment:
Small detail, but would it make more sense to grab all the jobs in one go around line 385? Otherwise we're hitting the db for each job unnecessarily.